packages/smooth_app/lib/pages/onboarding/onboarding_flow_navigator.dart

```diff
@@ -123,4 +130,17 @@ class OnboardingFlowNavigator {
       ),
     );
   }
+
+  static void getHistoryOnboradingNav(OnboardingPage currentPage) {
+    _historyOnboradingNav.add(currentPage);
+  }
+
+  static bool isOnboradingPagedInHistory(OnboardingPage page) {
```

Comment:
Could you please add a short comment to the code on what this method and `_historyOnboradingNav`  is for

Reply:
✅